pages/s/[org]/[repo]/index.tsx

```diff
@@ -55,6 +55,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   // Cache for two hours
   context.res.setHeader("Netlify-CDN-Cache-Control", "public, max-age=0, stale-while-revalidate=7200");
+  context.res.setHeader("Netlify-Vary", "query=range");
+  context.res.setHeader("Cache-Tag", `repo-pages,repo-page-${repoData.id}`);
```

Comment:
I added this because it will give us the ability to purge all repository pages from the cache (if needed) or on an individual basis. For more info about this see [Cache-tags & Purge API on Netlify](https://www.netlify.com/blog/cache-tags-and-purge-api-on-netlify/).